README.md

```diff
@@ -0,0 +1,48 @@
+This is a tool to download and create labeled audio datasets from YouTube videos.
+
+You need to execute two steps:
+
+### First step: download audio files and subtitles and convert audio format
+
+ - Add the YouTube video URLs to the `download.txt` file. The selected videos must have subtitles for the desired language.
```

Comment:
To be extra clear about it, you might want to specify that video URLs should be 'one per line'.

Reply:
Done